### DISCUSSION TITLE
What is the minimum required to run Upscayl?

### DISCUSSION THREAD

**Discussion by AkarinLiu** [upscayl/upscayl#491] [Q&A]
My computer is very old, my PC Information is:

```
Type: Lattop
Module: Acer Aspire3820TG
CPU: Intel Core i5 480M
GPU: AMD Radeon HD 5650M
```
Because my computer can`t run Upscayl, so I need your Support!
I know your will release Upscayl, but I want to deploy it on the cloudcompute platform for my privacy!
![image](https://github.com/upscayl/upscayl/assets/58081491/1cedd023-502e-4371-98d1-ef7c2fdd7ab6)

**Comment by aaronliu0130** [comment]
Did you try to set your GPU ID to that AMD integrated GPU as shown in the wiki? If it still doesn't work, do #390.

**Comment by lucas-0liveira** [comment]
I understand your concern about your old computer not being able to run Upscayl. It seems like you have a Lattop (assuming you meant "Laptop") model Acer Aspire3820TG with an Intel Core i5 480M CPU and an AMD Radeon HD 5650M GPU. Since your computer can't handle Upscayl, you're looking for support.

You mentioned that you're aware Upscayl will be released, but you'd like to deploy it on a cloud computing platform for privacy reasons. That's a reasonable request, and using a cloud platform can indeed provide more processing power and potentially better performance.

Is there anything specific you'd like to know or discuss about deploying Upscayl on a cloud computing platform? I'd be happy to assist you further.
